paddle/fluid/framework/device_worker.h

```diff
@@ -212,6 +212,7 @@ class DeviceWorker {
   FetchConfig fetch_config_;
   bool use_cvm_;
   bool no_cvm_;
+  bool scale_sparse_grad_;
```

Comment:
scale_sparse_gradient_with_batch_size

Reply:
have changed scale_sparse_grad to scale_sparse_gradient_with_batch_size